src/app/components/wrapper-components/online-game-wrapper/online-game-wrapper.component.ts

```diff
@@ -568,4 +614,13 @@ export class OnlineGameWrapperComponent extends GameWrapper<MinimalUser> impleme
             this.gameEventsSubscription.unsubscribe();
         }
     }
+
+    public override async getConfig(): Promise<MGPOptional<RulesConfig>> {
+        const rulesConfig: RulesConfig = this.configRoom.rulesConfig;
+        if (this.gameComponent != null) {
+            this.gameComponent.config = MGPOptional.of(rulesConfig);
+        }
```

Comment:
un getter qui est en fait un setter déguisé, peut-être mettre à jour son nom ?

Reply:
Comment t'appellerais cette fonction alors ?
Sachant que dans d'autres wrapper elle est différente

Reply:
je checkerai quand j'aurais la branche localement pour les quelques changements, histoire d'avoir une meilleure visibillité sur le code. 
Idéalement on voudrait que `.config` soit setté dès la création du jeu. Et cette fonction ci devrait juste retourner `this.configRoom.rulesConfig`, mais je verrai donc !

Reply:
this.gameComponent.config est setté à la config par défaut dans GameWrapper
mais dans getConfig on prend la version mentionné dans le document crée par PartCreation en ligne !
C'pas double usage!

Reply:
ok, donc si le but c'est de prendre la config choisie par part-creation, ça devrait pas être fait À CHAQUE FOIS QU'ON VEUT LIRE LA CONFIG :D 
Y'a des endroits potentiellement mieux : onGameStart, startPart, startGame (qui lui a la config room, donc il a accès à la config), ou ngOnInit

Reply:
Voilà j'ai commit ça en le bougeant dans startGame, tous les tests sont verts et le test manuel semble indiquer que c'est tout aussi bon !